buildenv/jenkins/Jenkinsfile

```diff
@@ -5,7 +5,7 @@ pipeline {
 		choice (choices: 'x64_linux\nx64_mac\ns390x_linux\nppc64le_linux\naarch64_linux', description: 'PLATFORM?', name: 'PLATFORM')
 		choice (choices: 'SE90\nSE80', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
 		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8', description: 'What is JVM Version?', name: 'JVM_VERSION')
-		choice (choices: 'openjdk\nsanity\nextended\nruntest', description: 'What is test level?', name: 'TARGET')
+		choice (choices: 'openjdk\nsanity\nextended\ntest_systemtest\nruntest', description: 'What is test level?', name: 'TARGET')
```

Comment:
for now this is fine, to leverage the ability to run directories of tests via a make target structured as test_dirName, shortly we will want to update this as we will add a make target called 'system'.